mmdet/models/necks/hrfpn.py

```diff
@@ -0,0 +1,87 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torch.utils.checkpoint import checkpoint
+
+from ..registry import NECKS
+
+
+@NECKS.register_module
+class HRFPN(nn.Module):
+
+    def __init__(self,
+                 in_channels,
+                 out_channels,
+                 normalize=None,
+                 pooling='AVG',
+                 share_conv=False,
+                 with_cp=False):
+        super(HRFPN, self).__init__()
+        assert isinstance(in_channels, list)
+        self.in_channels = in_channels
+        self.out_channels = out_channels
+        self.num_ins = len(in_channels)
+        self.with_bias = normalize is None
+        self.share_conv = share_conv
+        self.reduction_conv = nn.Sequential(
+            nn.Conv2d(in_channels=sum(in_channels),
```

Comment:
We can use `ConvModule` instead of `nn.Conv2d` to support flexible conv and norm types.